tests/test_constants.py

```diff
@@ -234,6 +234,11 @@ async def test_max_message_length(self, bot, chat_id):
             return_exceptions=True,
         )
         good_msg, bad_msg = await tasks
+
+        if isinstance(good_msg, BaseException):
```

Comment:
what if `bad_msg` is the one with the exception? or does return_exceptions always return to the first variable? Also, how are we running into this issue with a simple `send_message` only on this test but not on others (using the same `.gather`)?

Reply:
>  Also, how are we running into this issue with a simple send_message only on this test but not on others (using the same .gather)?

as far as I see, we're not usang `return_exceptions` in asyncio.gather in oher places

> what if bad_msg is the one with the exception? or does return_exceptions always return to the first variable?

gather retains the order, that should suffice